doc/helpers/responsive-extension.md

```diff
@@ -0,0 +1,127 @@
+```
+
+### XAML
+```xml
+xmlns:utu="using:Uno.Toolkit.UI"
+...
+
+<utu:ResponsiveExtension />
+```
+
+### Inheritance 
+Object &#8594; MarkupExtension &#8594; ResponsiveExtension
+
+### Constructors
+| Constructor           | Description                                                    |
+|-----------------------|----------------------------------------------------------------|
+| ResponsiveExtension() | Initializes a new instance of the `ResponsiveExtension` class. |
+
+### Properties
+| Property   | Type             | Description                                                                                         |
+| ---------- | ---------------- | --------------------------------------------------------------------------------------------------- |
+| Narrowest  | object           | Value to be used when the screen size is at its narrowest.                                          |
+| Narrow     | object           | Value to be used when the screen size is narrow.                                                    |
+| Normal     | object           | Value to be used when the screen size is normal.                                                    |
+| Wide       | object           | Value to be used when the screen size is wide.                                                      |
+| Widest     | object           | Value to be used when the screen size is at its widest.                                             |
+| Layout     | ResponsiveLayout | Overrides the MaxWidth for each screen size: `Narrowest`, `Narrow`, `Normal`, `Wide`, and `Widest`. |
+
+### Usage
+```xml
+<!-- Include the following XAML namespace to use the sample below -->
+xmlns:utu="using:Uno.Toolkit.UI"
+...
+<Page.Resources>
+	<SolidColorBrush x:Key="NarrowRed">Crimson</SolidColorBrush>
+	<SolidColorBrush x:Key="WideBlue">Blue</SolidColorBrush>
+</Page.Resources>
+...
+<Border Width="30"
+		Height="30"
+		HorizontalAlignment="Left"
+		Background="{utu:Responsive Normal={StaticResource NarrowRed},
+									Wide={StaticResource WideBlue}}" />
+```
+
+## Remarks
+**Platform Limitations**: The ability to update property values when the window size changes is only available on targets other than Windows UWP. On UWP targets, where only the parameterless constructor is available, the property value can only be set initially and cannot be updated if the window size changes. This is a limitation of UWP, and the class logs a warning message when the extension is used on UWP.
+
+**Initialization**: The `ResponsiveHelper` needs to be hooked up to the window�s `SizeChanged` event in order for it to receive updates when the window size changes. This is typically done in the `App.xaml.cs` file, where you can get the current window and call the `HookupEvent` method on the `ResponsiveHelper`. It�s important to do this when the app launches, otherwise the `ResponsiveExtension` won�t be able to update the property values when the window size changes.
```

Comment:
```suggestion
**Initialization**: The `ResponsiveHelper` needs to be hooked up to the window's `SizeChanged` event in order for it to receive updates when the window size changes. This is typically done in the `App.xaml.cs` file, where you can get the current window and call the `HookupEvent` method on the `ResponsiveHelper`. It's important to do this when the app launches, otherwise the `ResponsiveExtension` won't be able to update the property values when the window size changes.
```
avoid french symbols